public/stylesheets/style.css

```diff
@@ -103,3 +103,7 @@ button, #login-in-btn {
 input[type=checkbox]:checked + label.strikethrough{
   text-decoration: line-through;
 }
+
+.test-list.list {
```

Comment:
should be just `.list {`
Now you are trying to style element with class list which is a child of .test-list